finetuner/constants.py

```diff
@@ -35,6 +35,7 @@
 MODEL = 'model'
 MODEL_OPTIONS = 'model_options'
 ARTIFACT_ID = 'artifact_id'
+__DEFAULT_TAG_KEY__ = 'finetuner_label'
```

Comment:
no need for `__` here. Lets be consistent with the rest. `DEFAULT_TAG_KEY = 'finetuner_label'`